internal/commands/create.go

```diff
@@ -58,6 +63,7 @@ Create constraints with the Gatekeeper enforcement action set to dryrun
 	cmd.PersistentFlags().StringP("output", "o", "", "Specify an output directory for the Gatekeeper resources")
 	cmd.PersistentFlags().BoolP("dryrun", "d", false, "Sets the enforcement action of the constraints to dryrun, overriding the @enforcement tag")
 	cmd.PersistentFlags().Bool("skip-constraints", false, "Skip generation of constraints")
+	cmd.PersistentFlags().Bool("generate-v1", false, "Generate apiVersionv1 objects")
```

Comment:
nit: I think this flag is a bit ambiguous. It would be more clear if this was a string flag along the lines of `constraint-template-version` with a default of `v1beta1` and an updated help text to match.

Reply:
Done like that